### ISSUE TITLE
Warning with the latest version of react.js

### ISSUE THREAD

**Issue by djaonourside** [elsassph/haxe-react-redux#5]
I have the warning in a browser console: "Accessing PropTypes via the main React package is deprecated. Use the prop-types package from npm instead." React library version is 15.5.4.

**Comment by elsassph** [comment]
Yes this is fine until React 16. Haxe-react has been refactored to expose `react.ReactPropTypes` instead of `react.React.PropTypes` and a simple haxe-react update will be done to point to the now distinct `proptypes` npm module.

**Comment by djaonourside** [comment]
elsassph, thanks for explanation and your haxe smart version of react-redux:)
